### PR TITLE
Added the Pull Request Temple.

### LINKED ISSUE #10: Need to create pull request template file for repository.
## Need to create pull request template file.

I think it is not difficult for a beginner or not a beginner. I am too lazy to do this, so I hope someone takes this not a difficult task.

You can see the pull request template in other repositories (it is not in all repositories), or if you know how it should look, then go for it!

If there are questions related to this which I do not know, ask them here and we will discuss this.

Info about creation pull request file https://help.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository

### PR DESCRIPTION
**Pull request by huangyz0918**
closes #10 

Please review, thanks @MairwunNx .

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Closes #
```
Comment (MairwunNx) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: 🤔 What is `Closes # Addresses #`?

Comment (MairwunNx) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: Understand. :)

**Review by MairwunNx — APPROVED** [review]
## Changes approved 🎉 Thanks for contributing! Happy hacking 🍀!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (huangyz0918) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: Yes, contributors need to noted that which issues are related and which issues can be addressed here.

**Pull request merged**